### ISSUE TITLE
Polar V800 strava segment upload support

### ISSUE THREAD

**Issue by Antash** [cmaion/polar#3]
Strava live segments for v800 and m460 are out now, but unfortunately premium account require ma make them work.
It would be outstanding to support manual live segments uploading to make them work without having strava premium account.

I currently started implementation, you can check my fork and contact me for details if you want to join development.

**Comment by cmaion** [comment]
Hi

Nice work, thanks for the heads up!

It seems that you have the upload working? that was something I wanted to add at some point but never took the time to investigate the wire protocol for that. Glad you did!

I'll keep an eye on your fork :)

Cheers

**Comment by Antash** [comment]
Yes, upload is working now. There are a lot of embedded limitations (eg. it is not allowed to create ROUTES folder on the m4xx devices which are not supporting routes) and some small communication issues but in general it works!
